### PR TITLE
[ADD] stock_picking_report_incoming_delivery_address

### PR DESCRIPTION
**Pull request by EmilioPascual**
Sometimes in incoming transaction types it is necessary to show the delivery address, such as returns. If carrier has to pick up the goods at the address where they were delivered.

https://www.loom.com/share/a375c0ed66cb4fc991f09e1a077fe6b7?sid=0b3d2403-1991-48ec-8118-b6764f4c3cbe

@yajo @Shide @Gelojr @fcvalgar @rafaelbn @rousseldenis please review.

MT-5963 @moduon

### CODE REVIEW

File: stock_picking_report_incoming_delivery_address/readme/CONFIGURE.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+To configure this module, you need to:
+
+1. Go to Inventory App > Configuration > Operation Type.
+1. Select incoming operation type like returns o create new.
```
Comment (yajo) [inline stock_picking_report_incoming_delivery_address/readme/CONFIGURE.md]:
```suggestion
1. Select incoming operation type like returns or create new.
```

**Review by yajo — CHANGES_REQUESTED** [review]
it seems the address you should change in the report is the pickup one, not the delivery one.

**Comment by EmilioPascual** [comment]
I've update behavior, now pick-up address is shown in delivery slip and picking operations if check **Show pick-up address in report** is activated in operation type.

https://www.loom.com/share/e3336dc4f3ed4ecaa75fe479c69bee0e?sid=80ea144e-8e55-4077-a5e7-df1aad891434

**Review by yajo — APPROVED** [review]
This is a lot more code than the previous version, but it really fixes the problem. Good work!

**Review by Shide — CHANGES_REQUESTED** [review]
**nitpick:** Both report should have the same arrange of fields to maintain consistency and show the display_name of the pick-up address (commercial_partner, name)

Picking Operations
image

Delivery Slip
image

**Comment by EmilioPascual** [comment]
> **nitpick:** Both report should have the same arrange of fields to maintain consistency and show the display_name of the pick-up address (commercial_partner, name)

I don't understand what you mean

**Review by Gelojr — APPROVED** [review]
Nice Job. I don't add another video because yours is enough. LGTM

**Review by Shide — APPROVED** [review]
As talked with Emilio, Odoo does this as we see

**Review by rafaelbn — APPROVED** [review]
Lovely tested in runboat with 😄 !

👍🏼 

Thank you!

/ocabot merge patch

**Pull request merged**